tinker/e_announcements/e_announcements_controller.py

```diff
@@ -229,3 +230,29 @@ def get_search_results(self, selection, title, date):
                 to_return.append(annz)
 
         return to_return, forms_header
+
+    def get_title_and_message(self, form, ea_display):
+        title = find(form, 'title', False)
+        message = find(form, 'message', False)
+        ea_id = find(form, 'id', False)
+        created = find(form, 'created-on', False)
+        ea_display.append({
+            'title': title,
+            'message': message,
+            'id': ea_id
+        })
+
+    @cache.memoize(timeout=599)
```

Comment:
Its probably better to keep these numbers even. So setting it to 600 is better than 599. :)